navigation/src/jsMain/kotlin/com/huanshankeji/androidx/navigation/compose/NavHost.js.kt

```diff
@@ -56,30 +57,31 @@ actual fun NavHost(
 ) {
 
     //val lifecycleOwner = LocalLifecycleOwner.current
-    //val viewModelStoreOwner = LocalViewModelStoreOwner.current ?: rememberViewModelStoreOwner()
+    val viewModelStoreOwner = LocalViewModelStoreOwner.current ?: rememberViewModelStoreOwner()
 
-    //navController.setViewModelStore(viewModelStoreOwner.viewModelStore)
+    navController.setViewModelStore(viewModelStoreOwner.viewModelStore)
 
     // Then set the graph
     navController.graph = graph
 
+    // This seems not needed here since it seems also related to animations and transitions. (comment added when adapting from Compose UI `NavHost)
```

Comment:
```suggestion
    // This seems not needed here since it seems also related to animations and transitions. (comment added when adapting from Compose UI `NavHost`)
```